lib/telemeter/patch.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright:: Copyright (c) 2018 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Telemetry
+  def dead_deliver(data = {}) # this breaks telemetry tests
```

Comment:
iirc we had to monkey patch this because we wanted our config-based  checks for telemetry opt-in, while Telemetry's `Decision` only knows about the env vars/files at specific locations. 

We should be able to safely remove this patch, particularly if we go forward with https://github.com/chef/chef-telemetry/issues/15